.github/workflows/release.yaml

```
@@ -77,126 +81,199 @@ jobs:
+
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: macos-assets-${{ matrix.platform }}
+          path: ${{ github.workspace }}/dist/carveracontroller-community-*.dmg
+          retention-days: 2
 
   build-linux-artifacts:
+    #uncomment below to disable this job for testing
+    #if: false
     name: build-linux-${{ matrix.platform }}
     runs-on: ${{ matrix.image }}
     strategy:
       fail-fast: false
       matrix:
-        platform: [ x64 ]  # github hosted arm runners not standard yet
+        platform: [x64,arm64]
         include:
           - platform: x64
-            image: ubuntu-20.04
-          # - platform: aarch64
-          #   image: ubuntu-20.04
+            image: ubuntu-24.04
+          - platform: arm64
+            image: ubuntu-24.04-arm
     defaults:
       run:
         shell: bash
     steps:
-    - uses: actions/checkout@v4
-
-    - name: Set up Python 3.12
-      uses: actions/setup-python@v5
-      with:
-        python-version: 3.12
-
-    - name: Install poetry and dependencies
-      uses: ./.github/actions/bootstrap-poetry
-      with:
-        os: linux
-    
-    - name: Install MTDev
-      run: sudo apt-get install -y mtdev-tools gettext
-    
-    - name: Generate version strings
-      run: |
-        poetry self add "poetry-dynamic-versioning[plugin]"
-        poetry dynamic-versioning
-
-    - name: Build release
-      run: xvfb-run poetry run python scripts/build.py --os linux
-
-    - name: Rename artifact
-      run: poetry run bash ./scripts/rename-release-file.sh linux
-
-    - name: Upload artifact
-      uses: actions/upload-artifact@v4
-      with:
-        name: linux-assets
-        path: ${{ github.workspace }}/dist/carveracontroller-community-*.appimage
-        retention-days: 2
+      - uses: actions/checkout@v4
+
+      - name: Set up Python 3.12
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.12
+
+      - name: Install poetry and dependencies
+        uses: ./.github/actions/bootstrap-poetry
+        with:
+          os: linux
+
+      - name: Install MTDev
+        run: sudo apt-get install -y mtdev-tools gettext
+
+      - name: Generate version strings
+        run: |
+          poetry self add "poetry-dynamic-versioning[plugin]"
+          poetry dynamic-versioning
+
+      - name: Build release
+        run: xvfb-run poetry run python scripts/build.py --os linux
+
+      - name: Rename artifact
+        run: poetry run bash ./scripts/rename-release-file.sh linux
+
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: linux-assets-${{ matrix.platform }}
+          path: ${{ github.workspace }}/dist/carveracontroller-community-*.appimage
+          retention-days: 2
 
   build-pypi-artifacts:
+    #uncomment below to disable this job for testing
+    #if: false
     name: build-python
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Probably nothing wrong with using the latest, but for consistency with the other Linux build sake I prefer to use the same image. Rather not have to troubleshoot two different issues etc.

```suggestion
    runs-on: ubuntu-20.04  # matching the other Linux runner images
```